x-pack/plugins/rollup/public/crud_app/store/actions/start_jobs.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createAction } from 'redux-actions';
+import { toastNotifications } from 'ui/notify';
+import { startJobs as sendStartJobsRequest } from '../../services';
+import { loadJobs } from './load_jobs';
+
+export const startJobsSuccess = createAction('START_JOBS_SUCCESS');
+export const startJobs = (jobIds) => async (dispatch) => {
+  try {
```

Comment:
![image](https://user-images.githubusercontent.com/1965714/44491639-c63a4900-a616-11e8-8b90-7ab1b5ee3ee8.png)

when I try to start multiple jobs with mixed statuses (1 already started, 1 stopped), I get an internal service error. sdout:
```
[exception] Cannot start task for Rollup Job [1] because state was [STARTED] :: {"path":"/_xpack/rollup/job/1/_start","query":{},"statusCode":500,"response":"{\"error\":{\"root_cause\":[{\"type\":\"exception\",\"reason\":\"Cannot start task for Rollup Job [1] because state was [STARTED]\"}],\"type\":\"exception\",\"reason\":\"Cannot start task for Rollup Job [1] because state was [STARTED]\"},\"status\":500}"}
```

stopping mixed jobs appears to work fine though!

Reply:
I get the error too but when I refresh the job shows up as started. I'll mention it to Zach, thanks!